wsmaster/che-core-api-workspace/src/test/resources/devfile/devfile.yaml

```diff
@@ -14,6 +14,7 @@
 apiVersion: 1.0.0
 metadata:
   name: petclinic-dev-environment
```

Comment:
is it valid to have both name and generateName ?

Reply:
it is valid, but 'generateName' is basically ignored in that case.